### PR TITLE
Add SFI 23

### PR DESCRIPTION
**Pull request by samplackett**
https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/213682

### CODE REVIEW

File: test/helpers/reset-database.js [hunk 1]
```diff
@@ -6,12 +6,13 @@ const vetVisits = require('../mocks/schemes/vet-visits')
 const cs = require('../mocks/schemes/cs')
 const bps = require('../mocks/schemes/bps')
 const fdmr = require('../mocks/schemes/fdmr')
+const sfi23 = require('../mocks/schemes/sfi23')
```
Comment (Kirsten1822) [inline test/helpers/reset-database.js:9]: I'd put this below manual just to match the order of the schemes

File: test/helpers/reset-database.js [hunk 2]
```diff
@@ -6,12 +6,13 @@ const vetVisits = require('../mocks/schemes/vet-visits')
...
 const manual = require('../mocks/schemes/manual')
 const { sfiHoldCategory, sfiPilotHoldCategory, manualLedgerHoldCategory, debtEnrichmentHoldCategory } = require('../mocks/holds/hold-category')
 
 const resetDatabase = async () => {
   await db.sequelize.truncate({ cascade: true })
-  await db.scheme.bulkCreate([sfi, sfip, lumpSums, vetVisits, cs, bps, fdmr, manual])
+  await db.scheme.bulkCreate([sfi, sfip, lumpSums, vetVisits, cs, bps, fdmr, sfi23, manual])
```
Comment (Kirsten1822) [inline test/helpers/reset-database.js:15]: I'd put this below manual just to match the order of the schemes

File: test/unit/processing/account-codes/maps/get-map.test.js [hunk 3]
```diff
@@ -20,6 +20,11 @@ describe('get map', () => {
     expect(map).toStrictEqual(sfiMap)
   })
 
+  test('should return SFI map for SFI 23', () => {
+    const map = getMap(SFI23)
+    expect(map).toStrictEqual(sfiMap)
+  })
+
```
Comment (Kirsten1822) [inline test/unit/processing/account-codes/maps/get-map.test.js:27]: Again I'd move this to below FDMR just to match the order of the schemes

File: test/helpers/reset-database.js (hunk 1 above)
Comment (samplackett) [reply test/helpers/reset-database.js]: order amended, thanks!

File: test/helpers/reset-database.js (hunk 2 above)
Comment (samplackett) [reply test/helpers/reset-database.js]: order amended, thanks!

**Comment by Kirsten1822** [comment]
I'd also add the ticket link to the description of this PR to keep us right

**Comment by samplackett** [comment]
> I'd also add the ticket link to the description of this PR to keep us right

Good spot, thanks! Added to the top of the PR.

**Review by Kirsten1822 — APPROVED** [review]
Happy to approve :)

**Pull request merged**